### PR TITLE
change anaconda.org to AS instance in err msg

### PR DESCRIPTION
**Pull request by PeterDSteinberg**
Fixes the error message that had anaconda.org hard-coded.  Refers to issue #205.

### CODE REVIEW

File: binstar_client/commands/login.py [hunk 1]
```diff
@@ -77,8 +78,13 @@ def interactive_get_token(args, fail_if_already_exists=True):
 
 
     if token is None:
-        msg = ('Sorry. Please try again '
-               '(go to https://anaconda.org/account/forgot_password '
+        if parsed_url.netloc.startswith('api.anaconda.org'):
+            netloc = 'anaconda.org'
+        else:
+            netloc = parsed_url.netloc
```
Comment (srossross) [inline binstar_client/commands/login.py]: In this case `parsed_url.netloc` will be something like `example.com/api` the `/api` should be removed.

Comment (PeterDSteinberg) [inline binstar_client/commands/login.py]: The netloc is only up to the first slash in the uri.

Comment (srossross) [inline binstar_client/commands/login.py]: Good point! Thanks

**Comment by srossross** [comment]
:+1: (It looks like the tests are passing, but the windows machines are offline)

**Comment by PeterDSteinberg** [comment]
Is this ready to merge?  I'm unsure of the dev processes on this repo.

**Comment by srossross** [comment]
Yes it looks ready to merge. It is the same as the process for anaconda-server.

**Pull request merged**